src/index.js

```diff
@@ -28,21 +28,24 @@ module.exports = async function(content, map) {
     // vue-loader, we need to read file to get the source code.
     const infoOrPromise = isSFC
       ? docgen.parse(this.resourcePath, options.docgenOptions)
-      : docgen.parseSource(content, this.resourcePath, options.docgenOptions)
+      : docgen.parseMulti(this.resourcePath, options.docgenOptions)
```

Comment:
@Aaron-Pool 
As [@shilman commented](https://github.com/pocka/vue-docgen-loader/pull/4#issuecomment-562456067), this line would cause an error because `parseMulti` does not exist in vue-docgen-api@3.x. Please add an existence check and fallback for `parseSource` (or `parse`).

Reply:
Right, sorry, I've done very little library development, so I'm not super familiar with how peer dependencies work. I can definitely do what you just described though 👍